README.md

```diff
@@ -189,6 +190,28 @@ Here's a list of supported alternative widgets for different JSONSchema data typ
 
 > Note: for numbers, `min`, `max` and `step` input attributes values will be handled according to JSONSchema's `minimum`, `maximium` and `multipleOf` values when they're defined.
 
+#### Hidden widgets
+
+It's possible to use an hidden widget for a given field by setting the `ui:widget` uiSchema directive to `hidden` for this field:
+
+```js
+const schema = {
+  type: "object",
+  properties: {
+    foo: {type: "boolean"}
+  }
+};
+
+const uiSchema = {
+  foo: {"ui:widget": "hidden"}
+};
+```
+
+> Notes
+>
+> - Hiding widgets is only supported for `boolean`, `string`, `number` and `integer` schema types;
+> - As hiding a field obviously prevents the user from entering data, the only way to set its value is to pass the form component a `formData` prop containing data for this field.
```

Comment:
I think this is a bit too obvious. You could just note *it takes its value from the `formData` prop...*

BTW this reminds me about an hidden auto-datefield, whose value could set from `onSubmit` :)